cmake/SetupPackages.cmake

```diff
@@ -120,13 +120,11 @@ endif ()
 if (RAJA_NEEDS_BLT_TPLS)
   if (NOT BLT_EXPORTED)
     set(BLT_EXPORTED On CACHE BOOL "" FORCE)
-  blt_import_library(NAME          blt_stub EXPORTABLE On)
-  set_target_properties(blt_stub PROPERTIES EXPORT_NAME blt::blt_stub)
-            install(TARGETS blt_stub
-                    EXPORT               bltTargets)
-    blt_export_tpl_targets(EXPORT bltTargets NAMESPACE blt)
-    install(EXPORT bltTargets
-      DESTINATION  lib/cmake/raja)
+    blt_import_library(NAME          blt_stub EXPORTABLE On)
+    set_target_properties(blt_stub PROPERTIES EXPORT_NAME blt::blt_stub)
+    install(TARGETS blt_stub
+            EXPORT  bltTargets)
+    blt_install_tpl_targets(lib/cmake/raja)
```

Comment:
@johnbowen42 should this be `blt_install_tpl_setups` instead?  All the non-Windows CI checks complain that `blt_install_tpl_targets` does not exist.  I don't see it in the BLT submodule either.

Reply:
@rhornung67 yes, absolutely. just a typo on my part

Reply:
Also this PR will not pass CI until this one is merged and the commit checked out in camp https://github.com/LLNL/camp/pull/138